infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -36,6 +36,7 @@
 
     private final Map<String, Object> opts;
     private final boolean isPersistent;
+    @Getter
```

Comment:
In the afternoon's discussion, move from central definition -> distributed close to actual use.